components/Modal/CreateBlueprint.js

```diff
@@ -176,7 +176,9 @@ class CreateBlueprint extends React.Component {
                       onBlur={(e) => this.handleErrors(e.target.value)}
                     />
                     {createBlueprint.errorNameVisible &&
-                      <span className="help-block"><FormattedMessage defaultMessage="A blueprint name is required." /></span>
+                      <span 
+                      className="help-block"
+                      id="help-block-msg"><FormattedMessage defaultMessage="A blueprint name is required." /></span>
```

Comment:
This might be a case where we should use `data-id` since the class `help-block` is being used for other messages that could display in the form.

I would just change `id` to `data-id` here.

Also, if you run `npm run lint`, it will probably complain about your indentation and tell you how to fix it.